src/flake8_pytest_style/plugins/parametrize.rs

```diff
@@ -134,23 +187,121 @@ fn check_names(checker: &mut Checker, expr: &Expr) {
                 if let Some(first) = elts.first() {
                     handle_single_name(checker, expr, first);
                 }
-            } else if names_type != types::ParametrizeNameType::Tuple {
-                checker.add_check(Check::new(
-                    CheckKind::ParametrizeNamesWrongType(names_type),
-                    Range::from_located(expr),
-                ));
+            } else {
+                match names_type {
+                    types::ParametrizeNameType::Tuple => {}
+                    types::ParametrizeNameType::List => {
+                        let mut check = Check::new(
+                            CheckKind::ParametrizeNamesWrongType(names_type),
```

Comment:
Do you mind adding these to the `fixable()` list in `registry.rs` (and the `commit()` list)?

Reply:
`ParametrizeNamesWrongType` is already in both lists. Is there another place where it should be added?

Reply:
Oh sorry, I was misled because the list didn't update -- but this is basically different cases of a rule that was already fixable _sometimes_, is that right?

Reply:
@charliermarsh that's correct. Not all cases were being fixed.